bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Bookie.java

```diff
@@ -1680,7 +1681,7 @@ public static void main(String[] args)
             buff.writeLong(1);
             buff.writeLong(i);
             cb.incCount();
-            b.addEntry(buff, cb, null, new byte[0]);
+            b.addEntry(buff, cb, null, new byte[0], LedgerType.FORCE_ON_JOURNAL);
```

Comment:
who uses this main()??

Reply:
@jvrao
I don't know

@sijie can we drop this ? maybe it is another issue

Reply:
@jvrao @eolivelli the existence of main function was used for testing, I think.